locales/bn/bn.json

```diff
@@ -569,7 +569,7 @@
     "Thailand": "থাইল্যান্ড",
     "Thanks for signing up! Before getting started, could you verify your email address by clicking on the link we just emailed to you? If you didn't receive the email, we will gladly send you another.": "সাইন আপ করার জন্য ধন্যবাদ! শুরু করার আগে, আমরা শুধু আপনার ইমেল লিঙ্কে ক্লিক করে আপনার ইমেল ঠিকানা যাচাই করতে পারে? আপনি ইমেইল পাবেন না করে থাকেন তাহলে, আমরা সানন্দে আপনাকে অন্য পাঠাতে হবে.",
```

Comment:
Is the correct character used to end a sentence?

In other places, I noticed the use of the vertical slash character (`|`), but here - a period (`.`).

This translation was by machine.